packages/docs/docs/core-api/data-schema.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Data Schema 
+---
+
+User-defined data schemas can be used when interfacing with the exposed `uint253` attestation data. This tool encodes and decodes attestation data, consolidating a user-defined attestation changes. 
```

Comment:
I'd replace this entry text with "The `DataSchema` class abstracts unirep data into a javascript object. This class can be used to encode and decode attestation data, and build attestations that are ready to be submitted to the unirep smart contract.

Reply:
Entry text added